huaweicloud/resource_huaweicloud_cdn_domain.go

```diff
@@ -278,6 +278,10 @@ func resourceCdnDomainV1() *schema.Resource {
 						"force_redirect":             &forceRedirectAndCompress,
 						"compress":                   &forceRedirectAndCompress,
 						"cache_url_parameter_filter": &cacheUrlParameterFilter,
+						"ipv6_enabled": {
```

Comment:
suggest to rename it to `ipv6_enable` keep consistent with [subnet](https://registry.terraform.io/providers/huaweicloud/huaweicloud/latest/docs/resources/vpc_subnet#ipv6_enable)

Reply:
fixed